living-room/.events

```diff
@@ -0,0 +1 @@
+python note.py
```

Comment:
For students to reproduce this, they just have to `cd .` to re-enter their current working dir. This changes the narrative of the note slightly, but could we integrate it this way?

Also, would it make more sense to hide this file?